src/libraries/System.Reflection.DispatchProxy/src/System.Reflection.DispatchProxy.csproj

```diff
@@ -2,6 +2,8 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <!-- This library has been annotated to be AOT safe -->
```

Comment:
Nothing in System.Reflection.DispatchProxy works with native AOT. It is good that we have it annotated, but I would not call it AOT safe.